tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -316,11 +316,26 @@ module.exports = {
     waitForFileVisible: function (fileName) {
       const rowSelector = this.getFileRowSelectorByFileName(fileName)
       const linkSelector = this.getFileLinkSelectorByFileName(fileName)
+      this
+        .useXpath()
+        .waitForElementVisible(rowSelector)
+        .getAttribute(linkSelector, 'filename', function (result) {
+          this.assert.strictEqual(result.value, fileName, 'displayed file name not as expected')
+        })
+      return this.useCss()
+    },
+    /**
+     *
+     * @param {string} path
+     */
+    waitForFileWithPathVisible: function (path) {
+      const rowSelector = this.getFileRowSelectorByFileName(path)
+      const linkSelector = this.getFileLinkSelectorByFileName(path)
       this
         .useXpath()
         .waitForElementVisible(rowSelector)
         .getAttribute(linkSelector, 'innerText', function (result) {
-          this.assert.strictEqual(result.value.trim(), fileName, 'displayed file name not as expected')
+          this.assert.strictEqual(result.value.trim(), path, 'displayed file name not as expected')
         })
       return this.useCss()
     },
```

Comment:
both ```waitForFileVisible``` and ```waitForFileWithPathVisible``` looks very much similar, except for the trim() part. Can we make them one function?